dev-python/python-jsonrpc-server/python-jsonrpc-server-0.3.2.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="A Python 2 and 3 asynchronous JSON RPC server "
+HOMEPAGE="https://github.com/palantir/python-jsonrpc-server"
+SRC_URI="https://github.com/palantir/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="examples"
+
+DEPEND="dev-python/future[${PYTHON_USEDEP}]
```

Comment:
With python packages, the correct is 99 % that this is a RDEPEND and not DEPEND ;)

Upstream enables python tests apparently, could you implement that in the ebuild?